packages/roosterjs-editor-core/lib/undo/UndoSnapshots.ts

```diff
@@ -2,22 +2,39 @@
 // to keep size under limit. This is kept at 10MB
 const MAXSIZELIMIT = 10000000;
 
+/**
+ * The UndoSnapshots data structure
+ */
 export default class UndoSnapshots {
     private snapshots: string[];
     private totalSize: number;
     private currentIndex: number;
 
+    /**
+     * Create an instance of UndoSnapshots
+     * @param maxSize The maximum stack size
+     */
     constructor(private maxSize: number = MAXSIZELIMIT) {
```

Comment:
I notice that you didn't add signature for all the constructor functions.

Reply:
Added signature for all the constructor functions.